api/.gitignore

```diff
@@ -0,0 +1 @@
+dist/
```

Comment:
Can we put this in the root directory gitignore instead? And use `*/dist` if needed

Reply:
done